src/pages/nav.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import {
+  MDBNavbar,
+  MDBNavbarBrand,
+  MDBNavbarNav,
+  MDBNavItem,
+  MDBNavLink,
+  MDBNavbarToggler,
+  MDBCollapse,
+  MDBDropdown,
+  MDBDropdownToggle,
+  MDBDropdownMenu,
+  MDBDropdownItem,
+  MDBContainer,
+  MDBBtn,
+} from 'mdbreact';
+
+const NavbarPage = () => {
```

Comment:
Change this to a normal function for consistence reasons only
```suggestion
function NavbarPage() {
```